### PR TITLE
MAUI-1459-Updated Scheduler timeslot, month views API's breaking, and added volume 1 2022 integrated features for scheduler in the user guide.

### PR DESCRIPTION
**Pull request by Karthickmani97**
Updated Scheduler timeslot, month views API's breaking, and added volume 1 2022 integrated features for the scheduler in the user guide.

### CODE REVIEW

File: MAUI/Scheduler/agenda-view.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+title: Agenda view in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about the Agenda view feature of Syncfusion .NET MAUI Scheduler (SfScheduler) control and more.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Agenda view in .NET MAUI Scheduler (SfScheduler)
+
+The agenda view displays a list of scheduled appointments grouped by week, between set minimum and maximum dates, with the appointments from the current date displayed by default. When the `AppointmentsSource` property of `SfScheduler` is null, the agenda view will show only the month, week, and date headers.
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/agenda-view.md:12]: displays the events or appointments (normal appointments, all-day appointments, recurring appointments, and span appointments) in chronological order, grouped by date between the minimum and maximum dates and the current date displayed by default. When the `AppointmentsSource` property of `SfScheduler` is null, the agenda view will show only the month, week, and date headers.

File: MAUI/Scheduler/agenda-view.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+};
+// Setting the text style for the appointments.
+this.Scheduler.AppointmentTextStyle = appointmentTextStyle;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Day header customization
+
+The day header can be customized by using the `DayHeaderSettings` property of `AgendaView` in the `SfScheduler.`
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/agenda-view.md:60]: The agenda view day header

File: MAUI/Scheduler/header.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Header in .NET MAUI Scheduler (SfScheduler)
+
+You can learn about SfScheduler's header height, date format, and appearance support of the SfScheduler.
+
+## Customize header height in scheduler
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/header.md:14]: ShowNavigation arrows - today button ,

File: MAUI/Scheduler/load-on-demand.md [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+It allows you to load appointments on-demand for the visible dates. The `ShowBusyIndicator` property can be used to start and stop the loading indicator animation before and after appointments are loaded.
+
+These can be achieved in two ways,
+
+* QueryAppointments event
+* QueryAppointments command
+
+### QueryAppointments event
+
+This event occurs after the view of the scheduler or the visible dates are changed and also allows `AppointmentsSource` for the visible dates to be loaded in on-demand. Below is a list of the arguments
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/load-on-demand.md:25]: /// In the scheduler agenda view, the QueryAppointments event is used to load more appointments when the new month is loaded on view, whereas the new month appointments are added in scheduler AppointmentsSource.
        /// Other than agenda view, the scheduler AppointmentsSource can be reset for a new visible date range to improve appointment loading performance.

File: MAUI/Scheduler/load-on-demand.md [hunk 5]
```diff
@@ -0,0 +1,296 @@
...
+{% highlight c# %}
+
+this.Scheduler.View = SchedulerView.Week;
+this.Scheduler.QueryAppointments += OnSchedulerQueryAppointments;
+
+private async void OnSchedulerQueryAppointments(object sender, SchedulerQueryAppointmentsEventArgs e)
+{
+    this.Scheduler.ShowBusyIndicator = true;
+    await Task.Delay(1500);
+    this.Scheduler.AppointmentsSource = this.GenerateSchedulerAppointments(e.VisibleDates);
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/load-on-demand.md:51]: Add agenda view code

File: MAUI/Scheduler/agenda-view.md (hunk 1 above)
Comment (Karthickmani97) [reply MAUI/Scheduler/agenda-view.md]: Updated.

File: MAUI/Scheduler/agenda-view.md (hunk 2 above)
Comment (Karthickmani97) [reply MAUI/Scheduler/agenda-view.md]: Updated.

File: MAUI/Scheduler/header.md (hunk 3 above)
Comment (Karthickmani97) [reply MAUI/Scheduler/header.md:14]: Updated.

File: MAUI/Scheduler/load-on-demand.md (hunk 5 above)
Comment (Karthickmani97) [reply MAUI/Scheduler/load-on-demand.md]: Updated.

File: MAUI/Scheduler/day-week-views.md [hunk 6]
```diff
@@ -264,9 +269,9 @@ private ObservableCollection<SchedulerTimeRegion> GetTimeRegion()
 
 ![recurring-exception-time-region-selection-restriction-in-timeslots-views-in-maui-scheduler](images/day-week-views/recurring-exception-time-region-selection-restriction-in-timeslots-views-in-maui-scheduler.png)
 
-### Customize special time region appearance
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/day-week-views.md:267]: Mention API breaking link to redirect in PR to follow up with website team

File: MAUI/Scheduler/agenda-view.md [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+title: Agenda view in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about the Agenda view feature of Syncfusion .NET MAUI Scheduler (SfScheduler) control and more.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Agenda view in .NET MAUI Scheduler (SfScheduler)
+
+The agenda view displays the events or appointments (normal appointments, all-day appointments, recurring appointments, and span appointments) in chronological order, grouped by date between the minimum and maximum dates and the current date displayed by default. When the `AppointmentsSource` property of `SfScheduler` is `null,` the agenda view will show only the month, week, and date headers.
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/agenda-view.md:12]: 1. `null,` -> `null`,
2. When the `AppointmentsSource` property of `SfScheduler` is `null,` the agenda view will show only the month, week, and date headers. **[ in mobile view and responsive UI] and for web view ?**

File: MAUI/Scheduler/agenda-view.md [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+    Subject = "Meeting",
+    StartTime = DateTime.Now,
+    EndTime = DateTime.Now.AddHours(1),
+    Background = Brush.Orange,
+});
+// Adding scheduler appointment into the AppointmentsSource.
+this.Scheduler.AppointmentsSource = appointments;
+
+{% endhighlight %}
+{% endtabs %}
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/agenda-view.md:43]: Change the order of the topics

1. Month Header,
2. Week Header
3. Day Header
4. Appointment text customization

File: MAUI/Scheduler/agenda-view.md [hunk 9]
```diff
@@ -0,0 +1,239 @@
...
+
+N> 
+* The default value of `DateFormat,` and `Height` are `MMM dd,` and `30` respectively.
+* For desktop UI, The agenda view displays the appointment only.
+
+## Month header customization
+
+The agenda month header view can be customized by using the `MonthHeaderSettings` property of `AgendaView` in the `SfScheduler.`
+
+### Customize Month header appearance using Style
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/agenda-view.md:174]: Month - use `month`

File: MAUI/Scheduler/agenda-view.md [hunk 10]
```diff
@@ -0,0 +1,239 @@
...
+
+this.Scheduler.AgendaView.MonthHeaderSettings.DateFormat = "MMM yyy";
+this.Scheduler.AgendaView.MonthHeaderSettings.Height = 200;
+this.Scheduler.AgendaView.MonthHeaderSettings.TextStyle = textStyle;
+this.Scheduler.AgendaView.MonthHeaderSettings.Background = Brush.LightGreen;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customize Month header appearance using DataTemplate
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/agenda-view.md:212]: Month - month

File: MAUI/Scheduler/appointments.md [hunk 11]
```diff
@@ -261,6 +263,40 @@ this.Scheduler.AppointmentsSource = appointment;
 N> 
 For example: From 12/13/2021 12:00AM to 12/14/2021 12:00AM.
 
+## Read only appointment
+
+A read-only appointment can be created with `IsReadOnly` property of the `SchedulerAppointment.` This property restricts the user's ability to edit and delete.
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/appointments.md:268]: This property restricts the user's ability to edit and delete. - this is wrong detail

File: MAUI/Scheduler/appointments.md [hunk 12]
```diff
@@ -788,7 +824,11 @@ N>
 
 N> [View sample in GitHub](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main/RecursiveExceptionAppointment/BusinessObject)
 
-## Customize appointment text style
+## Appointment appearance
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/appointments.md:827]: Appointment appearance - Appointment customization

File: MAUI/Scheduler/day-week-views.md [hunk 13]
```diff
@@ -309,26 +314,158 @@ private ObservableCollection<SchedulerTimeRegion> GetTimeRegion()
...
+    };
+
+    timeRegions.Add(timeRegion);
+    return timeRegions;
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customize special time region appearance using DataTemplateSelector
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/day-week-views.md:364]: DataTemplateSelector - why do we need to add this DataTemplateSelector for time region alone?

File: MAUI/Scheduler/header.md [hunk 14]
```diff
@@ -0,0 +1,198 @@
...
+                </DataTemplate>
+            </scheduler:SchedulerHeaderView.HeaderTemplate>
+        </scheduler:SchedulerHeaderView>
+    </scheduler:SfScheduler.HeaderView>
+ </scheduler:SfScheduler>
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customize header appearance using DataTemplateSelector
```
Comment (Nijamudeen-Mohamed) [inline MAUI/Scheduler/header.md:130]: DataTemplateSelector - any specific reason?

**Review by Nijamudeen-Mohamed — CHANGES_REQUESTED** [review]
Update my comments

File: MAUI/Scheduler/agenda-view.md (hunk 7 above)
Comment (Karthickmani97) [reply MAUI/Scheduler/agenda-view.md]: Updated content for the desktop view as well.

File: MAUI/Scheduler/agenda-view.md (hunk 8 above)
Comment (Karthickmani97) [reply MAUI/Scheduler/agenda-view.md:43]: Re-ordered the topics.

File: MAUI/Scheduler/agenda-view.md (hunk 9 above)
Comment (Karthickmani97) [reply MAUI/Scheduler/agenda-view.md]: Updated.

File: MAUI/Scheduler/agenda-view.md (hunk 10 above)
Comment (Karthickmani97) [reply MAUI/Scheduler/agenda-view.md]: Updated and ensured in all subtitles

File: MAUI/Scheduler/appointments.md (hunk 11 above)
Comment (Karthickmani97) [reply MAUI/Scheduler/appointments.md]: Updated proper details

File: MAUI/Scheduler/appointments.md (hunk 12 above)
Comment (Karthickmani97) [reply MAUI/Scheduler/appointments.md]: Updated as `Appointment customization`

File: MAUI/Scheduler/day-week-views.md (hunk 13 above)
Comment (Karthickmani97) [reply MAUI/Scheduler/day-week-views.md:364]: Added DateTemplateSelector for all related properties and explained performance issues under notes.

File: MAUI/Scheduler/header.md (hunk 14 above)
Comment (Karthickmani97) [reply MAUI/Scheduler/header.md:130]: The DataTemplateSelector code has been added in Ug based on their corresponding API's example codes, some of the APIs didn't have DataTemplateSelector example codes. So neglected here too.

But, now added DateTemplateSelector for all related properties and explained performance issues under notes.

File: MAUI/Scheduler/agenda-view.md [hunk 15]
```diff
@@ -0,0 +1,301 @@
...
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Agenda view in .NET MAUI Scheduler (SfScheduler)
+
+The agenda view displays the events or appointments (normal appointments, all-day appointments, recurring appointments, and span appointments) in chronological order, grouped by date between the minimum and maximum dates and the current date displayed by default. When the `AppointmentsSource` property of `SfScheduler` is `null`, the agenda view will show only the month, week, and date headers for mobile view and responsive UI, and for desktop UI it will display only the date header of display date.
+
+A agenda view displays different UI for mobile and desktop, for mobile it displays the month header, the week header, and the date header however for desktop, it displays the appointment only.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/agenda-view.md:14]: A agenda view displays different UI for mobile and desktop, for mobile it displays the month header, the week header, and the date header, whereas for desktop, it displays the appointment only.

File: MAUI/Scheduler/agenda-view.md [hunk 16]
```diff
@@ -0,0 +1,301 @@
...
+            </scheduler:SchedulerAgendaView.MonthHeaderTemplate>
+        </scheduler:SchedulerAgendaView>
+    </scheduler:SfScheduler.AgendaView>
+ </scheduler:SfScheduler>
+
+{% endhighlight %}
+{% endtabs %}
+
+N> 
+* The default value of `DateFormat,` and `Height` are `MMMM yyyy,` and `150` respectively.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/agenda-view.md:113]: The default values for `DateFormat` and `Height` are `MMMM yyyy` and `150`, respectively.

File: MAUI/Scheduler/agenda-view.md [hunk 17]
```diff
@@ -0,0 +1,301 @@
...
+* For desktop UI, The agenda view displays the appointment only.
+
+### Customize month header appearance using DataTemplateSelector
+
+You can customize the month header appearance by using the `MonthHeaderTemplate` property of `AgendaView` in the `SfScheduler.` The `DataTemplateSelector` can choose a `DataTemplate` at runtime based on the value of a data-bound to month header by using the `MonthHeaderTemplate.` It lets to choose a different data template for each month header, customizing the appearance of a particular month header based on certain conditions.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/agenda-view.md:118]: It allows you to choose a different data template for each month header, as well as customize the appearance of a particular month header based on certain conditions.

File: MAUI/Scheduler/agenda-view.md [hunk 18]
```diff
@@ -0,0 +1,301 @@
...
+{% endhighlight %}  
+{% highlight c# %}
+
+this.Scheduler.View = SchedulerView.Agenda;
+
+{% endhighlight %} 
+{% endtabs %}
+
+N>
+* The default value of `DateFormat,` and `Height` are `MMMM yyyy,` and `150` respectively.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/agenda-view.md:174]: The default value for `DateFormat` and `Height` are `MMMM yyyy` and `150`, respectively.

File: MAUI/Scheduler/appointments.md [hunk 19]
```diff
@@ -261,6 +263,40 @@ this.Scheduler.AppointmentsSource = appointment;
 N> 
 For example: From 12/13/2021 12:00AM to 12/14/2021 12:00AM.
 
+## Read only appointment
+
+A read-only appointment can be created with `IsReadOnly` property of the `SchedulerAppointment.` If you enable this property, user will not be able to perform edit, and delete operations.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/appointments.md:268]: If you enable this property, the user will be unable to perform edit and delete operations.

File: MAUI/Scheduler/appointments.md [hunk 20]
```diff
@@ -825,6 +865,137 @@ this.Scheduler.AppointmentTextStyle = appointmentTextStyle;
...
+{% endhighlight %}  
+{% endtabs %}
+
+N>
+* By default, the `SchedulerAppointment` is set as the `BindingContext` for `AppointmentTemplate` for both `SchedulerAppointment` and custom data object in `AppointmentsSource.`
+* The Custom data object can be bound in `AppointmentTemplate` by using `SchedulerAppointment.Data.`
+
+### Customize appointment appearance using DataTemplateSelector
+
+You can customize the appointment appearance by using the `AppointmentTemplate` property of `DaysView` in the `SfScheduler.` The `DataTemplateSelector` can choose a `DataTemplate` at runtime based on the value of a data-bound to scheduler appointment appearance by using the `AppointmentTemplate.` It lets to choose a different data template for each appointment, customizing the appearance of a particular appointment based on certain conditions.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/appointments.md:920]: It allows you to choose a different data template for each appointment, as well as to customize the appearance of a particular appointment based on certain conditions.

File: MAUI/Scheduler/date-navigation-restriction.md [hunk 21]
```diff
@@ -125,6 +125,29 @@ private void OnButtonClicked(object sender, EventArgs e)
 {% endhighlight %}
 {% endtabs %}
 
+### Allow view navigation
+
+You can quickly navigate to the respective day or timeline day view by a tap on the month cell or view header of the following scheduler views such as a week, work week, month, agenda, timeline week, timeline work week, and timeline month views by using the `AllowViewNavigation` property of the `SfScheduler.`
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/date-navigation-restriction.md:130]: By using the `AllowViewNavigation` property of the `SfScheduler,` you can quickly navigate to the respective day or timeline day view by tapping on the month cell or view header of the following scheduler views: week, work week, month, agenda, timeline week, timeline work week, and timeline month views.

File: MAUI/Scheduler/header.md [hunk 22]
```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Header in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about to customize header height, date format and appearance of Syncfusion .NET MAUI Scheduler (SfScheduler) control and more.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/header.md:4]: Learn here all about how to customize header height, date format, and appearance of Syncfusion .NET MAUI Scheduler (SfScheduler) control and more.

File: MAUI/Scheduler/header.md [hunk 23]
```diff
@@ -0,0 +1,201 @@
...
+title: Header in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about to customize header height, date format and appearance of Syncfusion .NET MAUI Scheduler (SfScheduler) control and more. 
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Header in .NET MAUI Scheduler (SfScheduler)
+
+You can learn about SfScheduler's header height, date format, and appearance support of the SfScheduler.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/header.md:12]: You can learn about SfScheduler's header height, date format, and appearance support.

Please highlight the control names

File: MAUI/Scheduler/header.md [hunk 24]
```diff
@@ -0,0 +1,201 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+## Header customization
+
+The header can be customized by using the `HeaderView` property of the `SfScheduler.`
+
+### Customize header appearance using text style
+
+You can style the background color, and textStyle by using the properties such as `Background,` and `TextStyle` properties of `HeaderView.`
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/header.md:68]: You can style the background color and textStyle by using the properties such as `Background` and `TextStyle` properties of `HeaderView.`

File: MAUI/Scheduler/header.md [hunk 25]
```diff
@@ -0,0 +1,201 @@
...
+        </scheduler:SchedulerHeaderView>
+    </scheduler:SfScheduler.HeaderView>
+ </scheduler:SfScheduler>
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customize header appearance using DataTemplateSelector
+
+You can customize the header appearance by using the `HeaderTemplate` property of `HeaderView` in the `SfScheduler.` The `DataTemplateSelector` can choose a `DataTemplate` at runtime based on the value of a data-bound to scheduler header by using the `HeaderTemplate.` It lets to choose a different data template for each header, customizing the appearance of a particular header based on certain conditions.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/header.md:132]: It allows you to choose a different data template for each header, as well as to customize the appearance of a particular header based on certain conditions.

File: MAUI/Scheduler/load-on-demand.md [hunk 26]
```diff
@@ -0,0 +1,311 @@
+---
+layout: post
+title: Load appointments on demand in .NET MAUI Scheduler | Syncfusion
+description: Learn here all about to load appointments on demand from visible dates in Syncfusion .NET MAUI Scheduler (SfScheduler) control, its elements, and more.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/load-on-demand.md:4]: Learn here all about how to load appointments on demand from visible dates in Syncfusion .NET MAUI Scheduler (SfScheduler) control, its elements, and more.

File: MAUI/Scheduler/load-on-demand.md [hunk 27]
```diff
@@ -0,0 +1,311 @@
...
+
+# Load on demand in .NET MAUI Event Scheduler (SfScheduler).
+
+The Scheduler provides the capability to display an interactive view when the view is changed, or swipe between the views, and also to load appointments on-demand with a loading indicator, thereby improving loading performance when there are appointments range for multiple years.
+
+## Load appointments on demand
+
+It allows you to load appointments on-demand for the visible dates. The `ShowBusyIndicator` property can be used to start and stop the loading indicator animation before and after appointments are loaded.
+
+These can be achieved in two ways,
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/load-on-demand.md:18]: These can be achieved in two ways:

File: MAUI/Scheduler/load-on-demand.md [hunk 28]
```diff
@@ -0,0 +1,311 @@
...
+
+* QueryAppointments event
+* QueryAppointments command
+
+### QueryAppointments event
+
+This event occurs after the view of the scheduler or the visible dates are changed and also allows `AppointmentsSource` for the visible dates to be loaded in on-demand. Below is a list of the arguments
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/load-on-demand.md:25]: This event occurs when the view of the scheduler or the visible dates are changed, and it also allows the `AppointmentsSource` for the visible dates to be loaded in on-demand. The following is a list of the arguments:

File: MAUI/Scheduler/load-on-demand.md [hunk 29]
```diff
@@ -0,0 +1,311 @@
...
+
+* `Sender`: This contains the `SfScheduler` object.
+
+* `SchedulerQueryAppointmentsEventArgs`: This event will be performed on the view of the scheduler or the visible dates are changed, and you can see the details about the VisibleDates.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/load-on-demand.md:29]: This event will be performed on the view of the scheduler or when the visible dates are changed, and you can see the details about the VisibleDates.

File: MAUI/Scheduler/load-on-demand.md [hunk 30]
```diff
@@ -0,0 +1,311 @@
...
+
+    * `VisibleDates`: Gets the current visible dates of scheduler that is used to load the appointments to the `AppointmentsSource` on demand which is used to reduce the appointment fetching performance to render in current visible dates from large data source.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/load-on-demand.md:31]: from large data source--->from a large data source

File: MAUI/Scheduler/load-on-demand.md [hunk 31]
```diff
@@ -0,0 +1,311 @@
...
+        });
+    }
+
+    return appointments;
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+N> The `QueryAppointments` event will be raised if any one of the following action is performed,
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/load-on-demand.md:120]: The `QueryAppointments` event will be triggered, if any one of the following actions is performed.

File: MAUI/Scheduler/load-on-demand.md [hunk 32]
```diff
@@ -0,0 +1,311 @@
...
+* Once the `ViewChanged` event is raised, the `QueryAppointments` will be raised.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/load-on-demand.md:121]: * Once the `ViewChanged` event is triggered, the `QueryAppointments` will be triggered.

File: MAUI/Scheduler/load-on-demand.md [hunk 33]
```diff
@@ -0,0 +1,311 @@
...
+* If an appointment has been added or removed in the current visible dates, then the `QueryAppointments` event will not be triggered. Because the appointments for that visible date are already loaded.
+* In the scheduler agenda view, the `QueryAppointments` event is used to load more appointments when the new month is loaded on view, whereas the new month appointments are added in scheduler `AppointmentsSource.`
+* Other than agenda view, the scheduler `AppointmentsSource` can be reset for a new visible date range to improve appointment loading performance.
+
+### QueryAppointments command
+
+The Scheduler notifies the `QueryAppointmentsCommand,` when the view of the scheduler or the visible dates are changed. Get a visible dates from the `SchedulerQueryAppointmentsEventArgs.` The default value for this `ICommand` is null. The `SchedulerQueryAppointmentsEventArgs` passed as a command parameter.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/load-on-demand.md:128]: The Scheduler notifies the `QueryAppointmentsCommand,` when the view of the scheduler or the visible dates are changed. Get a list of visible dates from the `SchedulerQueryAppointmentsEventArgs.` The default value for this `ICommand` is null. The `SchedulerQueryAppointmentsEventArgs` is passed as a command parameter.

File: MAUI/Scheduler/load-on-demand.md [hunk 34]
```diff
@@ -0,0 +1,311 @@
...
+
+A ViewModel class should implement a command and handle it by the `CanExecute` and `Execute` methods to handle on-demand loading. In execute method, perform the following operations.
+
+The `ShowBusyIndicator` property can be used to start and stop the loading indicator animation before and after appointments are loaded into the `AppointmentsSource` of the `SfScheduler.`
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/load-on-demand.md:132]: The `ShowBusyIndicator` property can be used to start and stop the loading indicator animation, before and after appointments are loaded into the `AppointmentsSource` of the `SfScheduler.`

File: MAUI/Scheduler/load-on-demand.md [hunk 35]
```diff
@@ -0,0 +1,311 @@
...
+
+N> The `QueryAppointmentsCommand` will be raised if any one of the following action is performed,
+* Once the `ViewChanged` event is raised, the `QueryAppointmentsCommand` will be raised.
+* If an appointment has been added or removed in the current time visible date, then the `QueryAppointments` command will not be triggered. Because the appointments for that visible date are already loaded.
+* In the scheduler agenda view, the `QueryAppointments` event is used to load more appointments when the new month is loaded on view, whereas the new month appointments are added in scheduler `AppointmentsSource.`
+* Other than agenda view, the scheduler `AppointmentsSource` can be reset for a new visible date range to improve appointment loading performance.
+
+## Show busy indicator
+
+The `Scheduler` supports to show the busy indicator view by using the `ShowBusyIndicator` property of the `SfScheduler.` The default value is set to `false.` If the value is set to `true` and visible dates are changed then the busy indicator will be loaded on view.
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/load-on-demand.md:264]: If the value is set to `true` and visible dates are changed, then the busy indicator will be loaded on view.

File: MAUI/Scheduler/load-on-demand.md [hunk 36]
```diff
@@ -0,0 +1,311 @@
...
+{% highlight c# %}
+
+this.Scheduler.View = SchedulerView.Week;
+this.Scheduler.ShowBusyIndicator = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+N>  
+* You might start and stop the animation before and after the appointments loaded, when using the `QueryAppointments,` or `QueryAppointmentsCommand.`
```
Comment (laksmipriyamurugesan) [inline MAUI/Scheduler/load-on-demand.md:284]: When using the `QueryAppointments` or `QueryAppointmentsCommand,` you can start and stop the animation before and after the appointments load.

File: MAUI/Scheduler/appointments.md [hunk 37]
```diff
@@ -825,6 +865,137 @@ this.Scheduler.AppointmentTextStyle = appointmentTextStyle;
...
+    <Grid.Resources>
+        <DataTemplate x:Key="appointmentTemplate">
+            <Grid Background="BlueViolet">
+                <Label Text="{Binding Subject}" TextColor="White" HorizontalOptions="Center" VerticalOptions="Center" FontFamily="Bold"/>
+            </Grid>
+        </DataTemplate>
+    </Grid.Resources>
+    <scheduler:SfScheduler  x:Name="Scheduler" 
+                            View="Day"
+                            AllowedViews="Day,Week,WorkWeek,Month,Agenda,TimelineDay,TimelineWeek,TimelineWorkWeek,TimelineMonth">
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/appointments.md:885]: AllDayAppointmentTemplate , MoreAppointmentsTemplate add missed title

**Review by Jeyasri-Murugan — APPROVED** [review]

**Review by Nijamudeen-Mohamed — APPROVED** [review]
Fine

**Pull request merged**